internal/traffic/lb_test.go

```diff
@@ -41,11 +43,14 @@ func ExampleLoadBalancer_OpenSocket() {
 			if !ok {
 				return // chan is empty and closed.
 			}
-			_, _ = io.Copy(&w, r)
+			defer r.Close()
+			if _, err := io.Copy(&w, r); err != nil {
+				return
+			}
 		}
-		fmt.Println(w.String())
 	}
-	// Outputs: foo
+	// Outputs:
+	// foo
```

Comment:
I made this change just so i could point it out. this works either way.

Go examples are also tests. `// Outputs:` is special syntax that `go test` will look for and confirm that the example stdout matches the `// Outputs:` comment. 

https://go.dev/blog/examples